notebooks/scikit-learn/training.ipynb

```diff
@@ -0,0 +1,233 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Copyright 2019 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "# https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# scikit-learn Training\n",
+    "This notebook uses the [Census Income Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) to demonstrate how to train a model.\n",
+    "\n",
+    "\n",
+    "##  The data\n",
+    "The [Census Income Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) that this sample\n",
+    "uses for training is provided by the [UC Irvine Machine Learning\n",
+    "Repository](https://archive.ics.uci.edu/ml/datasets/). Google has hosted the data on a public GCS bucket `gs://cloud-samples-data/ml-engine/sklearn/census_data/`. \n",
+    "\n",
+    " * Training file is `adult.data`\n",
+    " * Evaluation file is `adult.test` (not used in this notebook)\n",
+    "\n",
+    "Note: Your typical development process with your own data would require you to upload your data to GCS so that you can access that data from inside your notebook. However, in this case, Google has put the data on GCS to avoid the steps of having you download the data from UC Irvine and then upload the data to GCS.\n",
+    "\n",
+    "### Disclaimer\n",
+    "This dataset is provided by a third party. Google provides no representation,\n",
+    "warranty, or other guarantees about the validity or any other aspects of this dataset.\n",
+    "\n",
+    "# Part 1: Create your python model file\n",
+    "\n",
+    "First, you'll create the python model (provided below). This is similar to your normal process for creating a scikit-learn model. However, there is one key difference:\n",
+    "1. Downloading the data from GCS at the start of your file, so that you can access the data.\n",
+    "\n",
+    "The code in this file loads the data into a pandas DataFrame that can be used by scikit-learn. Then the model is fit against the training data. Lastly, sklearn's built in version of joblib is used to save the model to a file that can be uploaded to [ML Engine's prediction service](https://cloud.google.com/ml-engine/docs/scikit/getting-predictions#deploy_models_and_versions)."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Looks good, do you think w ecan split this cell into 4 more:
1) Imports
2) GCS storage access
3) Model training
4) Export model

Reply:
Done. For the sectional comments, should I leave them as code comments or make them like text in the notebook?